pkg/controller/nodegroup/nodegroup_controller.go

```diff
@@ -402,3 +408,9 @@ func shouldUpdate(crdParams []*cloudformation.Parameter, cfnParams []*cloudforma
 	}
 	return false
 }
+
+func getSSMParamKey(version string) string {
+	eksOptimizedAMIKey := "/aws/service/eks/optimized-ami/<EKS_VERSION>/amazon-linux-2/recommended/image_id"
```

Comment:
nitpick: single use variable. i.e. we can directly use it in strings.Replace call without creating variable.